rust/shalom_dart_codegen/templates/macros.dart.jinja

```diff
@@ -1,17 +1,26 @@
 
 {% macro field_to_json_logic(fields) %}
 {% for name, field in fields | items %}
+    {% set input_ty = input_type(field.ty) %}
```

Comment:
dude stop with the "input_X" naming, OFC thats inside an input but its a **field** *of an input* naming is crutial for undesrtanding what you wrote later on. be mindful on them.. we should keep naming consistent.